### DISCUSSION TITLE
Wrong PRs number in Pull Shark badge

### DISCUSSION THREAD

**Discussion by MarcoBramini** [community/community#18247] [Other Feature Feedback, Questions, & Ideas]
Today the "Pull Shark" badge appeared in my profile. I got level 1, but I'm pretty sure i got more than 100  merged PRs . 😢
Is this because of merged PR for organizations I'm no more included?
Thank you!

EDIT: I just noticed that the badge correctly shows lvl 3, anyway the description shows "inaccessible". 
"Private organization" could be a better choice.

**Comment by Sulagna-Dutta-Roy** [comment]
Pull Shark badge only for private repositories contributions. Hope you will earn gold one badge someday.

**Comment by Vishu26** [comment]
I noticed the lvl grows as 2nd, 16th, 128th and 1024th pull request merge. You'll get the gold shark badge at 1024th pull request merge.

**Comment by devcrafta** [comment]
We can find out more about Badge here.
https://github.com/Schweinepriester/github-profile-achievements
